README.md

```diff
@@ -4,7 +4,10 @@
 [![Total Downloads](https://poser.pugx.org/nyholm/psr7/downloads)](https://packagist.org/packages/nyholm/psr7)
 [![Monthly Downloads](https://poser.pugx.org/nyholm/psr7/d/monthly.png)](https://packagist.org/packages/nyholm/psr7)
 [![Software License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](LICENSE)
-
+[![Roave](https://github.com/Nyholm/psr7/actions/workflows/bc.yml/badge.svg?branch=master)](https://github.com/Nyholm/psr7/actions/workflows/bc.yml?query=branch%3Amaster)
+[![Static analysis](https://github.com/Nyholm/psr7/actions/workflows/static.yml/badge.svg?branch=master)](https://github.com/Nyholm/psr7/actions/workflows/static.yml?query=branch%3Amaster)
+[![Tests](https://github.com/Nyholm/psr7/actions/workflows/tests.yml/badge.svg?branch=master)](https://github.com/Nyholm/psr7/actions/workflows/tests.yml?query=branch%3Amaster)
+[![Update branch alias](https://github.com/Nyholm/psr7/actions/workflows/branch-alias.yml/badge.svg)](https://github.com/Nyholm/psr7/actions/workflows/branch-alias.yml)
```

Comment:
I'd remove roave and this one, they are broken and don't provide any value to readers.

Reply:
Done. :heavy_check_mark: